bin/cyhy-domainsync

```diff
@@ -173,6 +175,12 @@ def main():
                         print("Created HostDoc owned by: " + default_owner)
                         continue
                     else:
+                        # If the --no-owned option is set and a HostDoc exists
+                        # that is owned by a non-default CyHy entity, skip this
+                        # IP address
+                        if args["--no-owned"] and host_doc["owner"] != default_owner:
+                            print("Skipping IP; --no-owned option enabled and found HostDoc owned by " + host_doc["owner"])
```

Comment:
Should we print the IP in the failure-related logging messages in this loop?  Would that be helpful to the users?

Reply:
The IP is already printed just before this message (see line 164):

https://github.com/cisagov/cyhy-core/blob/78781cbfe488d00f2dc55df3ee0e6d73adfa45c5/bin/cyhy-domainsync#L163-L184

So the output looks like this:
```
  1 IPv4 addresses found for this hostname
    192.168.10.10 Skipping IP; --no-owned option enabled and found HostDoc owned by TEST
```